book/website/communication/open/education.md

```diff
@@ -0,0 +1,82 @@
+    - While .pdf is an open format, it is also more closed in the sense that it is difficult to edit compared to formats such as RTF and HTML. 
+    - Accessibility is not guaranteed! 
+Learn more about how to improve the accessibility of your resources: 
+        - [How to Create Engaging Open Educational Resources without Sacrificing Accessibility](https://www.youtube.com/watch?v=UtOCI5TqtQI)
+        - [OER and Accessibility: Working Toward Inclusive Learning](https://sparcopen.org/news/2018/oer-accessibility-working-toward-inclusive-learning/)
+- Open [does not necessairly exclude revenue generation](https://edscoop.com/how-traditional-textbook-publishers-can-do-well-by-the-oer-community/)
+- Open Educational resources [may be more effective](http://openedgroup.org/review) than [traditional materials that are more expensive](https://www.huffpost.com/entry/college-textbooks-do-you_b_8261086). 
+See also {cite:ps}`Colvard2018OERsuccess`. 
+
+Learn more about why Open Education matters: 
+- '[Open Education Matters: Why is it important to share content?](https://www.youtube.com/watch?v=dTNnxPcY49Q)' 
+- '[Why Open Education Matters](https://www.youtube.com/watch?v=gJWbVt2Nc-I&list=PL741678F352148469)'
+- Open Education [benefits (and stakeholders)](https://openeducationalresources.pbworks.com/w/page/24838012/Stakeholders%20and%20benefits)
+
+(cm-open-education-promotion)=
+## Examples of promotion of Open Education
+- [Recognizing “Open” in Tenure and Promotion at University of British Columbia](https://sparcopen.org/news/2017/recognizing-open-tenure-promotion-ubc/) 
+- [What can university leaders do?](https://sparcopen.org/events/sparc-more-2016/what-university-leaders-can-do-to-promote-open-and-how-librarians-might-help/)
+
+
+(cm-open-education-oers)=
+# Open Educational Resources (OERs)
+
+Open educational resources (OERs) [{term}`def<Open Educational Resources>`] are teaching and learning materials that can be freely used and reused for learning or teaching at no cost, and without needing to ask permission. Examples are courses, including Massive Online Open Courses (MOOCs), lectures, teaching materials, assignments, and various other resources. OERs are available in many different formats compatible with online usage, most obviously text, images, audio, and video. Anyone with internet access can access and use OERs; access is not dependent on location or membership of a particular institution.
+
+Unlike copyrighted resources, OERs have been authored or created by an individual or organization that chooses to retain few, if any, ownership rights. In some cases, that means anyone can download a resource and share it with colleagues and students. In other cases, this may go further and enable people to edit resources and then re-post them as a remixed work. How do you know your options? OERs often have a Creative Commons licence or other permission to let you know how the material may be used, reused, adapted, and shared.
```

Comment:
Link to Creative Commons license website?

Reply:
Great idea, thanks!